### PR TITLE
[LLM] Add Arc demo gif to readme and readthedocs

### PR DESCRIPTION
**Pull request by Oscilloscope98**
Add Arc demo gif to readme and readthedocs

Main readme preview: https://github.com/Oscilloscope98/BigDL/tree/bigdl-llm-readme-update-arc#bigdl-llm-demos
LLM readme preview:  https://github.com/Oscilloscope98/BigDL/tree/bigdl-llm-readme-update-arc/python/llm#demos
Readthedocs preview: https://yuwentestdocs.readthedocs.io/en/bigdl-llm-readme-update-arc/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,13 +16,30 @@
...
      
 ### `bigdl-llm` Demos
-See the ***optimized performance*** of `chatglm2-6b`, `llama-2-13b-chat`, and `starcoder-15.5b` models on a 12th Gen Intel Core CPU below.
-
-<p align="center">
-          <a href="https://llm-assets.readthedocs.io/en/latest/_images/chatglm2-6b.gif"><img src="https://llm-assets.readthedocs.io/en/latest/_images/chatglm2-6b.gif" width='30%'></a> <a href="https://llm-assets.readthedocs.io/en/latest/_images/llama-2-13b-chat.gif"><img src="https://llm-assets.readthedocs.io/en/latest/_images/llama-2-13b-chat.gif" width='30%' ></a> <a href="https://llm-assets.readthedocs.io/en/latest/_images/llm-15b5.gif"><img src="https://llm-assets.readthedocs.io/en/latest/_images/llm-15b5.gif" width='30%' ></a>
-          <img src="https://llm-assets.readthedocs.io/en/latest/_images/llm-models3.png" width='76%'>
-</p>
-
+See the ***optimized performance*** of `chatglm2-6b` and `llama-2-13b-chat` models on a 12th Gen Intel Core CPU and a Intel Arc A770 Graphics below.
```
Comment (jason-dai) [inline README.md:19]: See the ***optimized performance*** of `chatglm2-6b` and `llama-2-13b-chat` models on 12th Gen Intel Core CPU and Intel Arc A770 GPU below

File: README.md [hunk 2]
```diff
@@ -17,12 +17,34 @@
...
      
 ### `bigdl-llm` Demos
-See the ***optimized performance*** of `chatglm2-6b`, `llama-2-13b-chat`, and `starcoder-15.5b` models on a 12th Gen Intel Core CPU below.
-
-<p align="center">
-          <a href="https://llm-assets.readthedocs.io/en/latest/_images/chatglm2-6b.gif"><img src="https://llm-assets.readthedocs.io/en/latest/_images/chatglm2-6b.gif" width='30%'></a> <a href="https://llm-assets.readthedocs.io/en/latest/_images/llama-2-13b-chat.gif"><img src="https://llm-assets.readthedocs.io/en/latest/_images/llama-2-13b-chat.gif" width='30%' ></a> <a href="https://llm-assets.readthedocs.io/en/latest/_images/llm-15b5.gif"><img src="https://llm-assets.readthedocs.io/en/latest/_images/llm-15b5.gif" width='30%' ></a>
-          <img src="https://llm-assets.readthedocs.io/en/latest/_images/llm-models3.png" width='76%'>
-</p>
+See the ***optimized performance*** of `chatglm2-6b` and `llama-2-13b-chat` models on 12th Gen Intel Core CPU and Intel Arc A770 GPU below.
```
Comment (jason-dai) [inline README.md:20]: change to `Intel Arc GPU`

File: README.md [hunk 3]
```diff
@@ -17,12 +17,34 @@
...
+
+<table width="100%">
+  <tr>
+    <td align="center" colspan="2">12th Gen Intel Core CPU</td>
+    <td align="center" colspan="2">Intel Arc A770 GPU</td>
```
Comment (jason-dai) [inline README.md:25]: Intel Arc GPU

Comment (Oscilloscope98) [reply README.md]: Have updated :)

**Review by jason-dai — APPROVED** [review]
LGTM

**Pull request merged**